app/routes/questionnaire.py

```diff
@@ -295,6 +295,10 @@ def get_thank_you(schema, session_store):
                     email=confirmation_email.get_url_safe_serialized_email(),
                 )
             )
+        else:
+            logger.info(
+                "email validation error", error_message=confirmation_email.form.errors
```

Comment:
The message needs to be extracted from `errors`, otherwise the contents of `error_message` will be something like:
```
{'email': [l'Enter an email address in a valid format, for example name@example.com']}
```

We should be able to use something like `str(confirmation_email.form.errors["email"][0])`. The `str` converts the lazy string into a plain string.